joplin/static/images/icon_down_arrow.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="12" height="7.4" viewBox="0 0 12 7.4"><defs><style>.a{fill:#1c1d1f;}</style></defs><g transform="translate(0 0)"><path class="a" d="M10.6,7.4,6,2.8,1.4,7.4,0,6,6,0l6,6Z" transform="translate(12 7.4) rotate(180)"/></g></svg>
```

Comment:
Is the main reason for including this vs using the wagtail arrow already provided for the animation purposes? 

In case a little code comment might be a nice breadcrumb for future devs

(you can see available icons here:
https://joplin-staging.herokuapp.com/admin/styleguide/)

Reply:
You got it!

Added a note to `applicaiton_banner.html`. And, yes the animation was the main reason. But, also this .svg from the design is a bit bolder than the built-in one, making it stand out a bit more.

Reply:
Also added that link to the Joplin_UI_Dev_Notes.md Notes!